src/fw/board/wscript_build

```diff
@@ -10,6 +10,8 @@ if board in ('bb2',):
             'drivers',
         ],
     )
+    bld.env.DEFINES.append('BT_VENDOR_ID=0x0154')
```

Comment:
Can these not just go in the board headers?

Reply:
actually yes, we can